### ISSUE TITLE
Solarized 256 colors not getting background color with mutt-kz and slang

### ISSUE THREAD

**Issue by psivesely** [altercation/mutt-colors-solarized#11]
I just compiled https://github.com/karelzak/mutt-kz (master branch, v1.5.23.1) against the S-Lang library because it seems mutt-colors-solarized don't work against it, but it hasn't help anything, at least not w/ 256 colors. (I never tested 16 colors on mutt w/ ncurses, so I can't say whether 16 colors was working before or not.)

I am sure my terminal (GNOME) supports 256 colors and tmux is configured correctly because everything looks great w/ the solarized 256 term colors and dircolors.

Do you think it's related to the fact mutt-kz adds the sidebar patch? Have you ever compiled w/ this option?

I could try to compile again w/ unpatched mutt.

For reference this is on Ubuntu 14.04 (LTS).

**Comment by kapunahelewong** [comment]
Hey, I don't know the answer to your question about S-Lang, but I am running Ubuntu 14.04 with Gnome and can confirm that 16 colors works but 256, I couldn't get to work.

**Comment by psivesely** [comment]
@altercation mentioned somewhere that you needed to compile mutt with S-Lang instead of ncurses if you want 256 colors to work. I did this and it still isn't working.

Can also confirm bug on Debian 8 GNOME terminal.

**Comment by dsaito** [comment]
I'm having the same issue. Solarized light 256 shows with a dark background. I'm using the brew version of mutt in Mac OSX 10.11.4.

**Comment by hegdevinayi** [comment]
@fowlslegs, you're right. The "compile vs SLang instead of ncurses" is in the *.muttrc files. I'm facing the same issues on Arch Linux (GNOME terminal/XTerm, within/without GNU Screen) for both light- and dark-256 colorschemes. Nothing seems to work.

**Comment by TornaxO7** [comment]
Does anyone found a solution to this?
